docs/ingestion/supervisor.md

```diff
@@ -24,30 +24,40 @@ sidebar_label: Supervisor
   -->
 
 A supervisor manages streaming ingestion from external streaming sources into Apache Druid.
-Supervisors oversee the state of indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained.
 
-This topic uses the Apache Kafka term offset to refer to the identifier for records in a partition. If you are using Amazon Kinesis, the equivalent is sequence number.
+Supervisors oversee the state of indexing tasks to coordinate [handoffs](../design/storage#indexing-and-handoff), manage failures, and ensure that the scalability and replication requirements are maintained.
 
-## Supervisor spec
+:::info
+This topic uses the Apache Kafka term _offset_ to refer to the identifier for records in a partition. If you are using Amazon Kinesis, the equivalent is _sequence number_.
+:::
 
-Druid uses a JSON specification, often referred to as the supervisor spec, to define streaming ingestion tasks.
-The supervisor spec specifies how Druid should consume, process, and index streaming data.
+Druid uses a JSON document, often referred to as a supervisor spec, to specify how Druid should consume, process, and index streaming data. A supervisor spec is a type of [native ingestion spec](../ingestion/ingestion-spec.md).
 
 The following table outlines the high-level configuration options for a supervisor spec:
 
 |Property|Type|Description|Required|
 |--------|----|-----------|--------|
-|`type`|String|The supervisor type. One of `kafka`or `kinesis`.|Yes|
-|`spec`|Object|The container object for the supervisor configuration.|Yes|
-|`spec.dataSchema`|Object|The schema for the indexing task to use during ingestion. See [`dataSchema`](../ingestion/ingestion-spec.md#dataschema) for more information.|Yes|
-|`spec.ioConfig`|Object|The I/O configuration object to define the connection and I/O-related settings for the supervisor and indexing tasks.|Yes|
-|`spec.tuningConfig`|Object|The tuning configuration object to define performance-related settings for the supervisor and indexing tasks.|No|
+|`type`|String|The type of ingestion to supervise, one of [`kafka`](kafka-ingestion.md) or [`kinesis`](kinesis-ingestion.md).|Yes|
+|[`spec`](#spec)|Object|The container object for the supervisor configuration.|Yes|
 |`suspended`|Boolean|Puts the supervisor in a suspended state|No|
 
-### I/O configuration
+## `spec`
```

Comment:
Hm, I am not sure if presenting the information this way makes it easer to comprehend. Personally, I think it's beneficial to include `dataSchema`, `ioConfig`, and `tuningConfig` in the high-level configurations table. Especially, since both `dataSchema` and `ioConfig` are required.  If you prefer to move them to a separate section, I think we should keep the existing descriptions and update the style of the bullet points. For example:

```suggestion
- `dataSchema`: Schema for the indexing task to use during ingestion.
- `ioConfig`: I/O configuration object to define the connection and I/O-related settings for the supervisor and indexing tasks.
- `tuningConfig`: Tuning configuration object to define performance-related settings for the supervisor and indexing tasks.
```